apiserver/facades/agent/caasoperator/operator.go

```diff
@@ -245,14 +245,14 @@ func (f *Facade) watchUnits(tagString string) (string, []string, error) {
 	return "", nil, watcher.EnsureErr(w)
 }
 
-// WatchUnitStart starts a StringWatcher to watch for Unit start events
-// on the CAAS api.
-func (f *Facade) WatchUnitStart(args params.Entities) (params.StringsWatchResults, error) {
```

Comment:
Might be worth retaining this method as deprecated and delete in 2.7.1
(since existing caas operators from 2.7-beta1 could call it).
Although it's only the unit init worker and any pods would almost certainly be running when the upgrade is done to this version. So probs is ok

Reply:
Discussed live, this was only introduced after beta1 so ok